app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const { getAllCategories } = require('./controllers/controller-games');
+
+const app = express();
+
+app.use(express.json());
```

Comment:
why is this here?

Reply:
Well, further down the line we are going to do POST and PATCH requests, so this is a method to handle that better.
I do agree that I should definitely JUST try to add the things specified in the ticket, especially when thinking about how actually coding in a team would go -> could cause merging conflicts etc.

I'll ensure to not get too far ahead of myself and instead just do the things one by one, ticket by ticket - thank you!